.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
 #        java: [ 8, 11, 17, 18, 19-ea ]
-        java: [ 8, 11, 17, 18]
```

Comment:
shouldn't this include jdk20 with this upgrade?

Reply:
@danischroeter The version of Gradle used doesn't work with jdk 20, and the gradle scalatest plugin doesn't work with newer versions.

Reply:
Ah right